SnakeLadder/src/Board.java

```diff
@@ -0,0 +1,48 @@
+import java.util.HashMap;
+import java.util.ArrayList;
+
```

Comment:
I see that the variables are not properly in camelCase at some places. Avoid that.

Reply:
I will take care of that